Dockerfile

```diff
@@ -35,7 +35,7 @@ COPY . .
 # Compile!
 RUN go build -ldflags="-buildid= -w" -trimpath -v -o /bin/mserv
 
-FROM debian:buster-slim AS runner
+FROM cgr.dev/chainguard/glibc-dynamic:latest AS runner
```

Comment:
presuming tested and tried. is there any way to ad run test when the workflow runs? one already?